src/test/java/com/amazon/ion/impl/EncodingDirectiveCompilationTest.java

```diff
@@ -136,6 +141,30 @@ private static void writeMacroSignature(IonRawWriter_1_1 writer, Map<String, Int
         writer.stepOut();
     }
 
+    private static void writeMacroSignatureFromDatagram(IonRawWriter_1_1 writer, Map<String, Integer> symbols, IonDatagram... signature) {
```

Comment:
This allows us to specify tagless encodings. The other variant just takes a `String`, without a way to add an encoding annotation.

Reply:
Fascinating use of IonDatagram for "unmodeled sequence of values". I did a double-take.